addCommas.js

```diff
@@ -0,0 +1,36 @@
+function addCommas(num) {
```

Comment:
fyi you can do bracket indexing (i.e. `"blah"[2]`) and `slice` on strings directly